twin.go

```diff
@@ -115,13 +165,23 @@ func (s *Substrate) CreateTwin(identity Identity, ip net.IP) (uint32, error) {
 }
 
 // UpdateTwin updates a twin
-func (s *Substrate) UpdateTwin(identity Identity, ip net.IP) (uint32, error) {
+func (s *Substrate) UpdateTwin(identity Identity, relay *string, pk *string) (uint32, error) {
```

Comment:
please change all to string and use pointer semantics later in the function body